scripts/spcgeonode/django/initialize.py

```diff
@@ -171,10 +178,14 @@
 print("-----------------------------------------------------")
 print("9. Securing GeoServer")
 
+# Default value are taken from geonode.settings
+geoserver_admin_username = os.getenv('GEOSERVER_ADMIN_USER', 'admin')
+geoserver_admin_password = os.getenv('GEOSERVER_ADMIN_PASSWORD', 'geoserver')
+
 # Getting the old password
 try:
-    r1 = requests.get('http://geoserver:8080/geoserver/rest/security/masterpw.json',
-                      auth=(admin_username, admin_password))
+    r1 = requests.get('{}/rest/security/masterpw.json'.format(_geoserver_host),
+                      auth=(geoserver_admin_username, geoserver_admin_password))
```

Comment:
GeoServer credentials might be different from Django admin credentials.
Allow to pass on this credentials to access GeoNode server.

As additional notes, use GEOSERVER_ADMIN_USER and GEOSERVER_ADMIN_PASSWORD as env name because it is already defined in the settings.py file.